globals.d.ts

```diff
@@ -0,0 +1,8 @@
+interface ObjectConstructor {
```

Comment:
Copied this from `coda`. Makes it so that `Object.keys()` is strongly typed, otherwise it's just `string[]` which is not helpful.